tests/lib/heap/prj.conf

```diff
@@ -1,3 +1,4 @@
 CONFIG_ZTEST=y
 CONFIG_SYS_HEAP_VALIDATE=y
 CONFIG_SYS_HEAP_RUNTIME_STATS=y
+CONFIG_SYS_HEAP_ALLOC_FREE_CALLBACKS=y
```

Comment:
These changes belong in commit `tests: lib/heap: add test for alloc/free callbacks`